tests/test_solvers/test_scikits_solvers.py

```diff
@@ -0,0 +1,138 @@
+#
+# Tests for the Scikits Solver class
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pybamm
+from pybamm.solvers.scikits_ode_solver import scikits_odes_spec
+
+
+import unittest
+import numpy as np
+
+
+class TestScikitsSolver(unittest.TestCase):
+
+    @unittest.skipIf(scikits_odes_spec is None,
+                     "scikits.odes not installed")
```

Comment:
I think you can skip the whole class by putting this on line 13 (avoids having to repeat for each method)